modules/log6x-config-roles.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// observability/logging/logging-6.0/log6x-clf.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="log6x-config-roles_{context}"]
+= Configuring Roles for Logging
+
+Logging does not grant all users access to logs by default. As an administrator, you must configure your users' access unless the Operator was upgraded and prior configurations are in place. Depending on your configuration and need, you can configure fine grain access to logs using the following:
```

Comment:
I tried to find where this content is in the preview and finally realized that the import is commented out.

I wanted to comment that I think it would make sense to clarify that this content is only relevant if the customer uses the LokiStack storage and (most probably) the visualization provided by the Cluster Observability Operator UIPlugin.

Maybe it does make sense to move this out into the COO documentation. There is a page documenting [the Logging UIPlugin](https://docs.openshift.com/container-platform/4.16/observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.html) now.

Reply:
@xperimental - Its commented out because its written in a way that's very 5.8 to 5.9 specific, and I believe that it overlaps with https://github.com/openshift/openshift-docs/pull/80999/files#diff-7e04c9b09b3e34b1e930ddc82c0164d718b8f3c68089681cab748a8e26aa4dcf - thoughts?

Reply:
The authorization scheme for giving users access to logs has not changed between 5.y and 6.0, so the documentation for "fine-grained logs access" is still valid. There are new steps related to RBAC that users need to do regarding the "allow the collector to collect logs" and "allow the collector to write to LokiStack", because, at least for the users that used the "default log store" this was done automatically by the CLO in 5.y .

The roles listed in `log6x-loki-rbac-rules-perms.adoc` are a separate set of roles again, which give access to the different custom resources the Loki Operator provides. As far as I'm aware they also already exist in 5.y .